sdk/tests/client/input_signing_data.rs

```diff
@@ -53,7 +53,7 @@ fn input_signing_data_conversion() {
         InputSigningData::try_from_dto(input_signing_data_dto.clone(), protocol_parameters.token_supply()).unwrap();
     assert_eq!(input_signing_data, restored_input_signing_data);
 
-    let input_signing_data_dto_str = r#"{"output":{"type":3,"amount":"1000000","unlockConditions":[{"type":0,"address":{"type":0,"pubKeyHash":"0x7ffec9e1233204d9c6dce6812b1539ee96af691ca2e4d9065daa85907d33e5d3"}}]},"outputMetadata":{"blockId":"0xedf5f572c58ddf4b4f9567d82bf96689cc68b730df796d822b4b9fb643f5efda","transactionId":"0xbce525324af12eda02bf7927e92cea3a8e8322d0f41966271443e6c3b245a440","outputIndex":0,"isSpent":false,"milestoneIndexBooked":0,"milestoneTimestampBooked":0,"ledgerIndex":0},"chain":{"coinType":4219,"account":0,"change":0,"addressIndex":0}}"#;
+    let input_signing_data_dto_str = r#"{"output":{"type":3,"amount":"1000000","unlockConditions":[{"type":0,"address":{"type":0,"pubKeyHash":"0x7ffec9e1233204d9c6dce6812b1539ee96af691ca2e4d9065daa85907d33e5d3"}}]},"outputMetadata":{"blockId":"0xedf5f572c58ddf4b4f9567d82bf96689cc68b730df796d822b4b9fb643f5efda","outputId":"0xbce525324af12eda02bf7927e92cea3a8e8322d0f41966271443e6c3b245a4400000","isSpent":false,"milestoneIndexBooked":0,"milestoneTimestampBooked":0,"ledgerIndex":0},"chain":{"coinType":4219,"account":0,"change":0,"addressIndex":0}}"#;
```

Comment:
But the serialisation should still be the 2 fields

Reply:
Not sure what you are asking but I think our type should have a `OutputId` yes. From it you can get the 2 fields, the reverse is annoying and fallible.